securedrop_client/app.py

```diff
@@ -46,18 +50,18 @@ def excepthook(*exc_args):
     sys.exit(1)
 
 
-def configure_logging():
+def configure_logging(home: str) -> None:
     """
     All logging related settings are set up by this function.
     """
-    if not os.path.exists(LOG_DIR):
```

Comment:
I was hoping to keep the directory logic all in one place so I pulled that out for now. I can readd it but...

I guess this brings up a broader question which is how strict do we want to be on data directory permissions? Like I think every dir needs to have `00` for the final two bytes, and we should bail otherwise. I know it's Qubes already, but if the directories don't need global `rwx`, then it shouldn't be set.

Reply:
let's keep the logs dir to prevent clutter

agreed on permissions